perception/radar_fusion_to_detected_object/README.md

```diff
@@ -0,0 +1,75 @@
+# radar_fusion_to_detected_object
+
+This package is radar-based sensor fusion module to 3d detected object.
+Main feature of this package are as following.
+
+- Attach velocity to lidar detection result from radar data to improve for tracking result and planning like adaptive cruise.
+- Improve detection result with radar sensor information. If both lidar 3d detected objects with low score and high confidence of radar pointcloud / objects, then improve score of objects.
+
+![process_low_confidence](docs/radar_fusion_to_detected_object_6.drawio.svg)
+
+## Core algorithm
+
+The document of core algorithm is [here](docs/algorithm.md)
+
+### Parameters for sensor fusion
+
+| Name                     | Type   | Description                                                                                            | Default value |
+| :----------------------- | :----- | :----------------------------------------------------------------------------------------------------- | :------------ |
+| bounding_box_margin      | double | The margin distance from bird's-eye view to choose radar pointcloud/objects within 3D bounding box [m] | 2.0           |
+| split_threshold_velocity | double | The threshold velocity to judge splitting two objects from radar information (now unused) [m/s]        | 5.0           |
```

Comment:
The object's velocity threshold is used to decide when to split radar objects (currently not implemented)

Reply:
fix

Reply:
Can you please write the commit where you fixed it?

Reply:
Fixed [here](https://github.com/scepter914/autoware.universe/commit/c8a936c536b64e7e80e8821db6cd9bd00c785a6d)